qiskit/providers/ibmq/api_v2/rest/schemas/job.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Schemas for job."""
+from marshmallow.validate import OneOf
+from qiskit.providers.ibmq.apiconstants import ApiJobStatus
+from qiskit.validation import BaseSchema
+from qiskit.validation.fields import String, Url, Nested, Number, Boolean
+
+
+# Helper schemas.
+
+class InfoQueueResponseSchema(BaseSchema):
+    """Nested schema for StatusResponseSchema"""
+
+    # Required properties
+    status = String(required=True)
+    position = Number(required=True, missing=0)
+
+
+class JobResponseSchema(BaseSchema):
+    """Nested schema for CallbackUploadResponseSchema"""
+    # pylint: disable=invalid-name
+
+    # Required properties
+    id = String(required=True)
+    kind = String(required=True)
+
+
+# Endpoint schemas.
+
+class PropertiesResponseSchema(BaseSchema):
+    """Schema for PropertiesResponse"""
+    pass
+
+
+class StatusResponseSchema(BaseSchema):
+    """Schema for StatusResponse"""
+
+    # Optional properties
+    infoQueue = Nested(InfoQueueResponseSchema, required=False)
```

Comment:
I don't know why some have `description` set and others don't, but this one could probably use one since it's not obvious what this property is for.

Reply:
According to @diego-plan9 I can only set descriptions *iff* there is already one in the docs or code. That's why you don't see them everywhere. I wanted to compose "my descriptions", but Diego asked me to remove them :) He mentioned that we will address them later